docs/cli.rst

```diff
@@ -292,6 +292,27 @@ See ``quickexact -h`` for a full list.
 
 The simulated ground state charge distribution can be printed with ``print -c``.
 
+ClusterComplete (``clustercomplete``)
+###########################
```

Comment:
use more ``#`` until the end of the line.

Reply:
Not sure how I missed that